docs/testing.rst

```diff
@@ -235,6 +237,9 @@ Pytest tooling has the following fixtures:
 Unittest
 ~~~~~~~~
 
+.. currentmodule:: aiohttp.test_utils
+
```

Comment:
Should it be two empty lines here?

Reply:
Maybe one empty line is enough but sorry, I don't want to wait for CI again before the merge.
Two lines are not such bad.
We have no strong rule for docs, sometimes it has one line as a delimiter but sometimes it has two or even more.